specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json

```diff
@@ -983,6 +983,7 @@
           "items": {
             "type": "string",
             "enum": [
+              "all",
```

Comment:
Are these real values the service supports? And is that on the data plane as well?

Reply:
I'm also concerned about whether this is a real value the service supports. Data plane doesn't seem to have it, unless I'm missing something.

Worth noting is that this PR most likely comes from this customer issue: https://github.com/Azure/azure-sdk-for-js/issues/10705